### PR TITLE
PRO-3087: Whitelist eligibility stop pages

### PR DESCRIPTION
**Pull request by bmj-vsalivendra**
**JIRA link (if applicable)**
https://tools.hmcts.net/jira/browse/PRO-3087

**Change description**
Whitelist all the stop pages for the eligibility questions
Remove "Save and Close" link from eligibility questions
Amend AD help block component tests

**Does this PR introduce a breaking change? (check one with "x")**
```
[ ] Yes
[X] No
```

### CODE REVIEW

**Comment by shepda** [comment]
I'll review these new changes but in future please don't add commits into a PR that aren't specifically to do with that PR.

**Comment by LucaDelBuonoHMCTS** [comment]
> @LucaDelBuono
> 
> I'll review these new changes but in future please don't add commits into a PR that aren't specifically to do with that PR.

Yeah I know, this got a bit out of hand, that' why I split it and created the other ticket about the AD component tests. Maybe review that first, once merged, this PR will be a lot smaller.

File: test/component/executors/testNotified.js [hunk 1]
```diff
@@ -30,8 +29,6 @@ describe('executor-notified', () => {
     });
 
     describe('Verify Content, Errors and Redirection', () => {
-        testHelpBlockContent.runTest('WillLeft');
```
Comment (shepda) [inline test/component/executors/testNotified.js]: This page displays the help block, so why don't we need this test?

File: test/component/executors/testRoles.js [hunk 2]
```diff
@@ -43,8 +42,6 @@ describe('executor-roles', () => {
     });
 
     describe('Verify Content, Errors and Redirection', () => {
-        testHelpBlockContent.runTest('WillLeft');
```
Comment (shepda) [inline test/component/executors/testRoles.js]: This page displays the help block, so why don't we need this test?

**Pull request merged**